src/cascadia/TerminalCore/terminalrenderdata.cpp

```diff
@@ -70,6 +70,12 @@ bool Terminal::IsCursorOn() const noexcept
     return cursor.IsOn();
 }
 
+void Terminal::SetCursorVisible(const bool isVisible) noexcept

```

Comment:
I'd prefer if this was in `Terminal.cpp` instead - I think this file is specifically for methods defined in the `IRenderData` interface (though that's not called out anywhere so that's my bad :P)